src/preferences/dialog/dlgprefeq.cpp

```diff
@@ -756,6 +756,11 @@ void DlgPrefEQ::slotMainEqEffectChanged(int effectIndex) {
                     QString valueText = QString::number((double)slider->value() / 100);
                     valueLabel->setText(valueText);
                     slidersGridLayout->addWidget(valueLabel, 2, i + 1, Qt::AlignCenter);
+
+                    // Immediately save the new (default) parameter values in preferences
```

Comment:
... because without 'Reset Parameters', the previously saved paramter values (of another EQ effect) would be loaded which (unnoticed) messes up the parameters shown directly after having switched the EQ effect.

Should I add that as comment?

Reply:
Yes please.

Reply:
Done. I adjusted the other comments, too